components/sup/src/manager/sys.rs

```diff
@@ -76,11 +85,17 @@ impl Sys {
         sys_info.set_hostname(self.hostname.clone());
         sys_info.set_gossip_ip(self.gossip_ip.to_string());
         sys_info.set_gossip_port(self.gossip_port as u32);
+        sys_info.set_ctl_gateway_ip(self.ctl_gateway_ip.to_string());
+        sys_info.set_ctl_gateway_port(self.ctl_gateway_port as u32);
         sys_info.set_http_gateway_ip(self.http_gateway_ip.to_string());
         sys_info.set_http_gateway_port(self.http_gateway_port as u32);
         sys_info
     }
 
+    pub fn ctl_listen(&self) -> SocketAddr {
+        SocketAddr::new(self.ctl_gateway_ip, self.ctl_gateway_port)
+    }
+
```

Comment:
Do you want `ctl_gateway_ip` and `ctl_gateway_port` to show up in the template data? If so, you'll need to tweak `templating::context::SystemInfo`, its `Serialize` implementation, the tests in `templating::context`, and the JSON schema in `components/sup/doc/render_context_schema.json` (as well as the documentation in `www/source/partials/docs/_reference-template-data.html.md.erb`, until @cnunciato unleashes his JSON-schema-to-documentation generator on the world, that is).

If you _don't_ want them in template data, then you don't have to do anything 😄

Reply:
Well, I mind as well! I'll update it